bears/general/KeywordBear.py

```diff
@@ -18,8 +18,12 @@ def _get_comments(dependency_results):
             not isinstance(annotation_bear_results, list)):
         return
 
-    for result in annotation_bear_results:
-        yield from result.contents.get('comments', [])
+    try:
+        for result in annotation_bear_results:
+            yield from result.contents.get('comments', [])
+    except AttributeError:
```

Comment:
@aptrishu I think it's a lot simpler to use github as the main review UI, just copying the text from gitter makes it way easier for other reviewers to see what's going on here